### PR TITLE
docs: typo fixes

### PR DESCRIPTION
**Pull request by fujitatomoya**
really minor fix only for docs.

I was reading the docs, and happened to see some typos could be addressed.

### CODE REVIEW

**Comment by Poorunga** [comment]
Welcome @fujitatomoya.

It seems that CI has failed. I see that you have modified the files in the vendor directory. The vendor directory is a third-party file downloaded through `go mod vendor`, and we should not modify it.

File: vendor/github.com/koron/go-ssdp/README.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ for {
 }
 ```
 
-### Send bye when quiting
+### Send bye when quitting
```
Comment (fujitatomoya) [inline vendor/github.com/koron/go-ssdp/README.md:49]: as @Poorunga mentioned, these are the vendor packages, should not be modified as this repo. see https://github.com/kubeedge/edgemesh/pull/461#issuecomment-1501912784

Comment (fujitatomoya) [reply vendor/github.com/koron/go-ssdp/README.md]: addressed by https://github.com/kubeedge/edgemesh/pull/461/commits/a9cbdcb31915aa4625e7141271da67c2747dcc6c

**Comment by fujitatomoya** [comment]
@Poorunga requesting another review.

**Pull request merged**